wordpress-mobile/WordPress-iOS
# Gutenberg: Fix RNAztec import hack.

This PR implements changes from [this PR](https://github.com/wordpress-mobile/react-native-aztec/pull/70) that fixes adding the RNAztec module as a dependency using Cocoapods.

This effectively fixes the hack used to build and run WPiOS and Guntenberg-mobile example app modifying manually an import statement.

**To test WPiOS:**
- Checkout this branch.
- `pod install`.
- Clean build folder.
- Run Metro inside `Gutenberg` sub dir.
- Build and run WPiOS.
- Check that there are no build errors.
- Open an instance of Gutenberg (i.e: start a new post).
- Check that the paragraph blocks work as expected.

**To test Gutenberg Example app:**
- Go (back) to `Gutenberg` sub dir.
- Run `yarn iOS`.
- Check that it builds successfully without the need of any change. 🎉

cc @diegoreymendez
